src/notification.rs

```diff
@@ -330,6 +343,44 @@ impl Notification {
         Ok(NotificationHandle::new(self.clone()))
     }
 
+     /// Sends Notification to NSUserNotificationCenter.
+    ///
+    /// Returns an `Ok` no matter what, since there is currently no way of telling the success of
+    /// the notification.
+    #[cfg(target_os = "windows")]
+    pub fn show(&self) -> Result<()> {
+        let sound_name = self.sound_name.clone();
```

Comment:
no clone necessary, try this:
```suggestion
        let sound = match &self.sound_name {
            Some(chosen_sound_name) => winrt_notification::Sound::from_str(&chosen_sound_name).ok(),
```